begrun.c

```diff
@@ -213,13 +217,16 @@ set_units(void)
     message(0, "UnitDensity_in_cgs = %g \n", All.UnitDensity_in_cgs);
     message(0, "UnitEnergy_in_cgs = %g \n", All.UnitEnergy_in_cgs);
     message(0, "Photon density OmegaG = %g\n",All.CP.OmegaG);
-    message(0, "Massless Neutrino density OmegaNu0 = %g\n",All.CP.OmegaNu0);
+    if(!All.MassiveNuLinRespOn)
```

Comment:
Perhaps we can start using '.' to create long parameter names. What is the relation between MassiveNuLinResp and HybridNeutrinos?

Reply:
You mean to organise the parameters for a module into a short struct, which is in turn a member of All? HybridNeutrinos is a sub-option of MassiveNuLinResp, indeed.

Reply:
At least a first clean up is the parameter names, `StarFormation.Enabled, StarFormation.Wind.Enabled, StarFormation.Wind.Model StarFormation.Wind; And neutrino goes to Neutrino.Enabled, Neutrino.Hybrid.Enabled, ... Using structs (like Cosmology) in the internal storage will ease later sharing the parser code between GenIC and Gadget too.

Reply:
I like this idea but I wonder if we should do it all at once, in a separate (later) pull. It is kind of strange when only the neutrino parameter obey this convention. Also it would be nice to have some kind of support for these structs from the param reading code - you had some idea to embed lua for reading parameters at one point, didn't you?

Reply:
OK. Let's plan for a massive parameter overhual later.